content/spin/kv-store.md

```diff
@@ -39,13 +39,13 @@ In all of these previous examples, to persist data within your application, a se
 
 However, from Spin v0.9.0 onwards, you are no longer required to install any on-disk persistence; outside of just installing Spin itself. SQLite exists inside Spin. SQLite is embedded in a way that is analogous to how one would imagine using a software library i.e. SQLite is designed to be used in this manner and therefore, for all intents and purposes, you could say that SQLite does not exist outside of Spin. When you run your Spin application using the [spin up](https://developer.fermyon.com/common/cli-reference#up) command SQLite is available to your application. When your application is no longer running, neither is SQLite. Your data will continue to persist at all times (including during restarts) and will support the running of your application.
 
-In the future, it may be possible to embed other database technologies into Spin (as v0.9.0 has shown here with SQLite). For example, future Spin SDK updates may target other databases such as Redis; which may allow you to build applications using Redis via minimal Spin configuration without the need to locally install and maintain your own Redis instance.
+In the future, it may be possible to embed other database technologies into Spin (as v0.9.0 onwards has shown here with SQLite). For example, future Spin SDK updates may target other databases such as Redis; which may allow you to build applications using Redis via minimal Spin configuration without the need to locally install and maintain your own Redis instance.
```

Comment:
The parenthesis here doesn't really add anything - could say `... into Spin, in the same way as SQLite.` if you want to keep something, but I'd drop "which version showed it".

Reply:
Parenthesis was removed, and it reads much better, thanks.